pprint/src/main/scala/ammonite/pprint/PPrint.scala

```diff
@@ -118,6 +131,21 @@ object PPrinter {
     s.toString()
   }
 
+  private def takeFirstLines(cfg: Config, iter: Iterator[String]):Iterator[String]={
+    @tailrec
+    def inner(lines: Int, iter: Iterator[String], begin: Iterator[String]): Iterator[String]={
+      if(!iter.hasNext || lines < 0) return begin
+      if(lines==0) return begin ++ Iterator(cfg.color.prefix("..."))
+      val head = iter.next()
+      if(lines < head.length/cfg.maxWidth){
+        return begin ++ Iterator(head.substring(0, lines*cfg.maxWidth), cfg.color.prefix("..."))
```

Comment:
Don't use `return` unless you really need to; e.g. here you can just have chained `if ... else if ... else if ... else`s